doc/reactive-migration.md

```diff
@@ -0,0 +1,45 @@
+---
+uid: Extensions.Migration.Reactive
+---
+
+# Upgrading Extensions Reactive Version
+
+## Upgrading to Extensions Reactive V5
+
+Upgrading to Uno.Extension.Reactive V5 should not require any changes to your application code. However, there are new features and improvements that you may want to take advantage of.
+
+### MVUX Generated Code
+
+As of V4, the MVUX analyzer generated a bindable proxy for each of the models in your app. For example, the bindable proxy for `MainModel` was named `BindableMainModel`.
+
+This behavior changed in V5, and the MVUX analyzer will now generate a ViewModel class for each of your app's models. For example, the generated code for `MainModel` is named `MainViewModel`.
```

Comment:
```suggestion
This behavior has changed in V5, and the MVUX analyzer will now generate a ViewModel class for each of your app's models. For example, the generated code for `MainModel` will be named `MainViewModel`.
```

Reply:
is `BindableMainModel` still there, or being replaced by `MainViewModel`?
the way it is word here, seems to be suggest we have both.
if that is not the case, you want to clarify that.

Reply:
to me, it is still not clear with the section below..?

Reply:
`BindableMainModel` is gone and instead we will have `MainViewModel` for the generated classes.

Maybe I am biased but I understand it from this sentence:

> This behavior has changed in V5, and the MVUX analyzer will now generate a ViewModel ...

maybe @agneszitte  could validate if it's something other would understand

Reply:
> You don't have to do anything if you update but want to keep the old behavior. The generator will continue to generate the bindable proxies.

suggests that **unless**, you tag the assembly with `[BindableGenerationTool(3)]`, we still have access to the legacy `BindableMainModel`

Reply:
That is correct. The new Version will not change since we default to version 2 of the BindableGenerationTool. But to update and get the latest generator features, users would need to use the `BindableGenerationTool(3)` assembly annotation

cc: @Xiaoy312